src/xmainloop.cpp

```diff
@@ -216,6 +216,10 @@ void XMainLoop::buttonpress(XButtonEvent* be) {
                     mm->mouse_initiate_move(client, {});
                 }
             }
+        } else {
+            // root windows handling
+            HSDebug("handle default RootWindow ButtonPress on win 0x%lx\n", be->window);
+            mm->mouse_call_command_root_window({"call", "spwan","xmenu.sh"});
```

Comment:
There's a typo `spawn` vs `spwan`. Also you're making a unencessary round-trip through the `mm` module. you can do the `callCommand` directly here:
```
std::ostringstream discardedOutput;
Input input("spawn", {"xmenu.sh"});
Command::call(input, discardedOutput);
```